storage/queue_test.go

```diff
@@ -87,7 +87,7 @@ func (s *StorageQueueSuite) TestQueueExists(c *chk.C) {
 	c.Assert(ok, chk.Equals, true)
 }
 
-func (s *StorageQueueSuite) TestPostMessage_PeekMessage_DeleteMessage(c *chk.C) {
+func (s *StorageQueueSuite) TestPostMessage_PeekMessage_PutMessage_DeleteMessage(c *chk.C) {
```

Comment:
Sure I agree. Happy to rename this. I was merely following the naming pattern of this test.
Name before I updated the test:  TestPostMessage_PeekMessage_DeleteMessage
After my change: TestPostMessage_PeekMessage_PutMessage_DeleteMessage

I can rename this to:
TestPutMessage_PeekMessage_UpdateMessage_DeleteMessage

Does this work?